map-structure/vi-map/src/vi-map.cc

```diff
@@ -38,6 +38,7 @@ VIMap::~VIMap() {}
 void VIMap::deepCopy(const VIMap& other) {
   clear();
   CHECK(mergeAllMissionsFromMapWithoutResources(other));
+  CHECK(checkMapConsistency(other));
```

Comment:
actually, wouldn't this check also need to be below, after the resource map deep copy?

Reply:
Yes, I agree!